apps/server/src/api/admin/users/post.ts

```diff
@@ -1,17 +1,29 @@
 import { ulid } from 'ulid';
-import { fixedGroups } from '@raise/shared';
 import { middyfy } from '../../../helpers/wrapper';
-import { assertHasGroup, insert } from '../../../helpers/db';
+import { insert, scan } from '../../../helpers/db';
 import { userTable } from '../../../helpers/tables';
 import { $Ulid, $UserCreation } from '../../../schemas';
+import { sendEmail } from '../../../helpers/email';
+import newUser from '../../../helpers/email/newUser';
 
 export const main = middyfy($UserCreation, $Ulid, true, async (event) => {
-  assertHasGroup(event, fixedGroups.National);
```

Comment:
We must still check the creator of the user is in the national team - I think without this check anyone on the platform can add new users, which is not the current intention. At the time, we thought keeping this control with the national team makes it easier to monitor use of the platform and reduce the risk and expected severity of security breaches.